lib/compilation-prepare.sh

```diff
@@ -0,0 +1,178 @@
+		[[ $(cat ${SRC}/cache/sources/${LINUXSOURCEDIR}/net/Kconfig | grep wireguard | wc -l) -gt 1 ]] && \
+		sed -i '0,/wireguard/{/wireguard/d;}' ${SRC}/cache/sources/${LINUXSOURCEDIR}/net/Kconfig
+		# headers workaround
+		display_alert "Patching WireGuard" "Applying workaround for headers compilation" "info"
+		sed -i '/mkdir -p "$destdir"/a mkdir -p "$destdir"/net/wireguard; \
+		touch "$destdir"/net/wireguard/{Kconfig,Makefile} # workaround for Wireguard' \
+		${SRC}/cache/sources/${LINUXSOURCEDIR}/scripts/package/builddeb
+
+	fi
+
+
+
+
+	# Wireless drivers for Realtek 8811, 8812, 8814 and 8821 chipsets
+
+	if linux-version compare $version ge 3.14 && [ "$EXTRAWIFI" == yes ]; then
+
+		# attach to specifics tag or branch
+		local rtl8812auver="branch:v5.2.20"
+
+		display_alert "Adding" "Wireless drivers for Realtek 8811, 8812, 8814 and 8821 chipsets ${rtl8812auver}" "info"
+
+		fetch_from_repo "https://github.com/aircrack-ng/rtl8812au" "rtl8812au" "${rtl8812auver}" "yes"
+		cd ${SRC}/cache/sources/${LINUXSOURCEDIR}
+		rm -rf ${SRC}/cache/sources/${LINUXSOURCEDIR}/drivers/net/wireless/rtl8812au
+		mkdir -p ${SRC}/cache/sources/${LINUXSOURCEDIR}/drivers/net/wireless/rtl8812au/
+		cp -R ${SRC}/cache/sources/rtl8812au/${rtl8812auver#*:}/{core,hal,include,os_dep,platform} \
+		${SRC}/cache/sources/${LINUXSOURCEDIR}/drivers/net/wireless/rtl8812au
+
+		# Makefile
+		cp ${SRC}/cache/sources/rtl8812au/${rtl8812auver#*:}/Makefile \
+		${SRC}/cache/sources/${LINUXSOURCEDIR}/drivers/net/wireless/rtl8812au/Makefile
+		cp ${SRC}/cache/sources/rtl8812au/${rtl8812auver#*:}/Kconfig \
+		${SRC}/cache/sources/${LINUXSOURCEDIR}/drivers/net/wireless/rtl8812au/Kconfig
+
+		# Adjust path
+		sed -i 's/include $(src)\/hal\/phydm\/phydm.mk/include $(TopDIR)\/drivers\/net\/wireless\/rtl8812au\/hal\/phydm\/phydm.mk/' \
+		${SRC}/cache/sources/${LINUXSOURCEDIR}/drivers/net/wireless/rtl8812au/Makefile
+		sed -i 's/include $(TopDIR)\/hal\/phydm\/phydm.mk/include $(TopDIR)\/drivers\/net\/wireless\/rtl8812au\/hal\/phydm\/phydm.mk/' \
+		${SRC}/cache/sources/${LINUXSOURCEDIR}/drivers/net/wireless/rtl8812au/Makefile
+
+		# Add to section Makefile
+		echo "obj-\$(CONFIG_RTL8812AU) += rtl8812au/" >> $SRC/cache/sources/${LINUXSOURCEDIR}/drivers/net/wireless/Makefile
+		sed -i '/source "drivers\/net\/wireless\/ti\/Kconfig"/a source "drivers\/net\/wireless\/rtl8812au\/Kconfig"' \
+		$SRC/cache/sources/${LINUXSOURCEDIR}/drivers/net/wireless/Kconfig
+
+	fi
+
+
+
+
+	# Wireless drivers for Realtek 8188EU 8188EUS and 8188ETV chipsets
+
+	if linux-version compare $version ge 3.14 && [ "$EXTRAWIFI" == yes ]; then
+
+		# attach to specifics tag or branch
+		local rtl8811euver="branch:v5.3.9"
+
+		display_alert "Adding" "Wireless drivers for Realtek 8188EU 8188EUS and 8188ETV chipsets ${rtl8811euver}" "info"
+
+		fetch_from_repo "https://github.com/aircrack-ng/rtl8188eus" "rtl8188eu" "${rtl8811euver}" "yes"
+		cd ${SRC}/cache/sources/${LINUXSOURCEDIR}
```

Comment:
wanna quote the vars here?

Reply:
Yes, you are welcome to sent it here at this branch.

Reply:
@mrueg are you planning on adding commits to this branch, or PR later?

Reply:
probably a PR later, apologies if you were waiting, didn't have time for that